### PR TITLE
Crud books

### PR DESCRIPTION
**Pull request by SipofTea**

### CODE REVIEW

File: controllers/books/new.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const book = require('../../models').book;
+module.exports = { 
+  add_book (req, res, next) {
```
Comment (karimabuseer) [inline controllers/books/new.js]: Maybe remove next

File: controllers/books/new.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+
+  add (req, res) {
+  return book
+    .create({
+      book_title: req.body.title,
+      book_author: req.body.author
+    })
+    .then(() => res.redirect("/books"))
+    .catch((error) => res.status(400).send(error));
```
Comment (karimabuseer) [inline controllers/books/new.js:16]: Maybe this would be a good basis to write a test

File: routes/books.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var router = express.Router();
+var bookList = require('../controllers/books/index.js')
+var addBook = require('../controllers/books/new.js')
+
+/* GET home page. */
+router.get('/', bookList.books);
```
Comment (karimabuseer) [inline routes/books.js:7]: Let's make this change to user too if we're surrounding by newlines.

**Review by karimabuseer — APPROVED** [review]

File: routes/books.js (hunk 3 above)
Comment (SipofTea) [reply routes/books.js:7]: Made comments to add white lines to users on users pr

**Pull request merged**